Core/include/Acts/Vertexing/AdaptiveMultiVertexFinder.hpp

```diff
@@ -318,12 +316,13 @@ class AdaptiveMultiVertexFinder {
   /// @param vtx The vertex candidate
   /// @param seedTracks The seed tracks
   /// @param fitterState The vertex fitter state
+  /// @param useBeamSpotConstraint Indicates whether beam spot constrain is used
   ///
   /// @return pair(nCompatibleTracks, isGoodVertex)
   std::pair<int, bool> checkVertexAndCompatibleTracks(
       Vertex<InputTrack_t>& vtx,
       const std::vector<const InputTrack_t*>& seedTracks,
-      FitterState_t& fitterState) const;
+      FitterState_t& fitterState, const bool& useBeamSpotConstraint) const;
```

Comment:
The reference is larger than the `bool`, so I think a by-value `bool` argument is fine.

```suggestion
      FitterState_t& fitterState, bool useBeamSpotConstraint) const;
```